### ISSUE TITLE
UnrealGT ActorInfo Crash 

### ISSUE THREAD

**Issue by Shromm-Gaind** [unrealgt/unrealgt#23]
Running UnrealGT on UE5.1 with win 10. 

The engine crashes when I try and use the ActorInfo uasset; 

[log.txt](https://github.com/unrealgt/unrealgt/files/10277144/log.txt)

Is anyone experiencing the same issue? It is showing it up as a memory problem, I have tried debugging the GTObjectFilter.cpp and the GTActorInfoGeneratorComponent.cpp using cout and flush however nothing pops on the terminal or when the engine crashes.

**Comment by StuartGJohnson** [comment]
Yeah, there is an infinite loop in dragonheat123's CanEditChange() fix. Looks like something changed in UE5.1 WRT this method. I have a possible fix which prevents the hang in the editor, but let me generate some goodies before getting to a PR.
